docs/conf/portland/2018/news/cfp-reminder.rst

```diff
@@ -0,0 +1,35 @@
+:template: 2018/na/generic.html
+
+.. post:: January 4, 2018
+   :tags: portland-2018, website, cfp, tickets
+
+
+Write the Docs Portland: CFP Reminder & Ticket Sales Open
+#########################################################
+
+Happy New Year, fellow documentarians!
+
+Stepping across the threshold into 2018, the Portland Write the Docs conference is suddenly feeling *much* closer! Our `call for proposals closes next Wednesday <http://www.writethedocs.org/conf/portland/2018/cfp/>`_, and `tickets for the conference are officially on sale <http://www.writethedocs.org/conf/portland/2018/tickets/>`_. Read on for more details on both!
+
+Submit your talk by Wednesday
+------------------------------------------------------------
+
+With the deadline for submissions coming up on **Wednesday, January 10th at midnight PST**, there's just a few days left to propose a talk for Write the Docs Portland!
+
+If you're interested or already working on pitching a talk, make sure you review the full `call for proposals <http://www.writethedocs.org/conf/portland/2018/cfp/>`_. And if you've already got an idea, but you’re having a hard time getting your proposal to come together, you can always ask for feedback on the `WTD Slack <http://slack.writethedocs.org/>`_. The community is there to help!
+
+As always, we're eagerly looking forward to your submissions! We’ll be reviewing all the talk proposals later this month, and announcing the full conference schedule in February.
+
+
+Get your ticket for the conference
+------------------------------------------------------------
+
+**The Portland conference will be returning to the Crystal Ballroom from May 6-8, 2018.** The `event website <http://www.writethedocs.org/conf/portland/2018/>`_ is up, with some preliminary details, and we'll be filling in more, as it gets closer. We're already excited to see what this year has in store!
+
+If you're excited too, it might interest you to know that `ticket sales are now live <http://www.writethedocs.org/conf/portland/2018/>`_! For the last several years the conference has sold out, so it doesn't hurt to buy early.
+
+One small change to make note of – we're switching up conference swag this year! Instead of our traditional t-shirts, your ticket will include a fresh new rendition of branded WTD swag. Stay tuned for more details!
```

Comment:
Will be curious if this gets much response. If we get a bunch of moaning, maybe it will tell us that people value them, heh.

Reply:
I had the same thought. We'll see! :)